pegasus/sites.v3/code.org/public/teacher-dashboard/review-hociyskvuwa/splat.haml

```diff
@@ -82,8 +82,8 @@ title: Review Hour of Code special events
 
   function submitReview(secret, review) {
     $.ajax({
-      url: "/v2/forms/HocSignup2015/" + secret,
-      type: "review",
+      url: "/v2/forms/HocSignup2015/" + secret + "/review",
```

Comment:
Note that we really ought to be URL encoding the secret.  Fortunately it doesn't contain any reserved characters.

No need to change that in the hotfix since the escaping is not currently needed and has always been lacking.